src/sch/eager.jl

```diff
@@ -24,7 +24,7 @@ function init_eager()
         sopts = SchedulerOptions(;allow_errors=true)
         opts = Dagger.Options((;scope=Dagger.ExactScope(Dagger.ThreadProc(1, 1)),
                                 occupancy=Dict(Dagger.ThreadProc=>0)))
-        Dagger.compute(ctx, Dagger.delayed(eager_thunk, opts)();
+        Dagger.compute(ctx, Dagger.delayed_internal(eager_thunk, opts)();
```

Comment:
Nitpicking, what do you think about naming it `_delayed()` instead? I know this literally has `internal` in the name but the leading underscore is also used in Python and `Base` so it might be a bit more recognizable.